controllers/helmchart_controller.go

```diff
@@ -460,6 +460,7 @@ func (r *HelmChartReconciler) buildFromHelmRepository(ctx context.Context, obj *
 		loginOpts []helmreg.LoginOption
 	)
 
+	normalizedURL := strings.TrimSuffix(repo.Spec.URL, "/")
```

Comment:
can we replace all occurences of `repo.Spec.URL` by `normalizedURL` in this method?

Reply:
good to me.